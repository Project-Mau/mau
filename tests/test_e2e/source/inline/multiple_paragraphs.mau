This document contains multiple paragraphs.

They are separated by an empty line.
