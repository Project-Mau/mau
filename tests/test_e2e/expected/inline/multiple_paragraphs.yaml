data:
  args: []
  content:
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: This document contains multiple paragraphs.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: They are separated by an empty line.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

